prov/gni/test/cq.c

```diff
@@ -91,6 +91,30 @@ void setup()
 	cq_attr.wait_obj = FI_WAIT_NONE;
 }
 
+void setup_1_4()
```

Comment:
This is exactly like the `setup` except the version, right?  Please just pass in the version as an argument to avoid duplication of the code.